app/src/main/java/com/eveningoutpost/dexdrip/Models/UserError.java

```diff
@@ -113,16 +113,6 @@ public static void cleanup() {
        new Cleanup().execute(deletable());
     }
 
-    public static void cleanup(long timestamp) {
```

Comment:
This function is still being consumed here: https://github.com/NightscoutFoundation/xDrip/blob/master/wear/src/main/java/com/eveningoutpost/dexdrip/ListenerService.java#L1152

If that logic is no longer relevant, can you remove it as part of this PR?

We also reference this function in a comment that we can remove in this PR: https://github.com/NightscoutFoundation/xDrip/blob/master/app/src/main/java/com/eveningoutpost/dexdrip/wearintegration/WatchUpdaterService.java#L957